### DISCUSSION TITLE
Product Status cannot be found error

### DISCUSSION THREAD

**Discussion by HEYY0M1KEY** [farag2/Sophia-Script-for-Windows#413] [Q&A] [answered]
I get the following error before I start. I included a screenshot from the sophia.psm1 Module file it is referencing. I did not make any changes to that file. I only made changes in the sophia.ps1. Appreciate any help.

![Sophia error](https://user-images.githubusercontent.com/117887608/201007069-d9e92e4d-16b9-4ed2-b933-827739603c40.PNG)

**Comment by farag2** [comment]
Hi. Unfortunately, it means that Defender eas removed and Windows is broken. :(
Did you remove Defender? A 3rd party AV installed doesn't remove it, we're talking about apps that let do it only.

**Reply by HEYY0M1KEY** [reply]
Fresh install and Windows Defender running. I'll see if I can't figure something out tomorrow. I'm going to run the Defender scan and see if that has anything to do with it, not sure why it would but can't hurt. Appreciate the response.

![image](https://user-images.githubusercontent.com/117887608/201016515-89ae6bf3-b48a-4b8c-99de-03aaa7d6595c.png)

**Comment by farag2** [comment, marked as answer]
What's your Windows version? May I check remotely to see what's going in via AnyDesk today?

**Reply by HEYY0M1KEY** [reply]
I’ll be available tomorrow, about 12 hours from now and can message. Windows 10 22H2.

**Reply by farag2** [reply]
Good. Cus this is strange message that hasn't to be visible.
Could you PM me to Telegram: @sanctuary_d?

**Reply by HEYY0M1KEY** [reply]
One day later a Windows Defender update came out. Installed the update and the error is gone. Thanks to farag for communicating and helping.
